lib/utils.robot

```diff
@@ -13,4 +18,36 @@ Ping Host
     [Arguments]     ${host}
     ${RC}   ${output} =     Run and return RC and Output    ping -c 4 ${host}
     Log     RC: ${RC}\nOutput:\n${output}
-    Should be equal     ${RC}   ${0}
\ No newline at end of file
+    Should be equal     ${RC}   ${0}
+
+Power On Host
+    log to console    "Powering on the host"
```

Comment:
I've made changes to this file that does what you want without sleeping.  See https://github.com/mkumatag/openbmc-automation/pull/51/commits/0eed501a5a8c7c3bab7ac6b3d35985cb114e00aa

I did not add Trigger Warm Reset so keep that one but delete the other 2 methods